packages/graph/deployment/graph/migrate_and_run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+set -euxo
+
+cd /migrations
+
+export HASH_GRAPH_PG_MIGRATION_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${HASH_GRAPH_PG_DATABASE}
+yarn graph:migrate up
+
+cd /
+
+export HASH_GRAPH_HOST="$POSTGRES_HOST"
+export HASH_GRAPH_ADDRESS="0.0.0.0:4000"
```

Comment:
nit/discussion
We also define this URI in the root `.env`. Don't know if we should share these somehow, the only difference is that the URI in the root splits ups the hostname/ip and the port, which makes them incompatible without interpolation, unfortunately

Reply:
Good point, used the environment variable instead: ce5381d19a496bfc4af09604ed074932b9f44835